env/production/performance.yaml

```diff
@@ -159,14 +167,14 @@ spec:
   - resource:
       name: cpu
       target:
-        averageUtilization: 25
```

Comment:
Are we confident that the system will still scale **up** appropriately? Do we want to test 50 on staging first?